src/Microsoft.Restier.Publishers.OData/Query/RestierQueryBuilder.cs

```diff
@@ -42,6 +42,7 @@ public RestierQueryBuilder(ApiBase api, ODataPath path)
             this.handlers[ODataSegmentKinds.EntitySet] = this.HandleEntitySetPathSegment;
             this.handlers[ODataSegmentKinds.Singleton] = this.HandleSingletonPathSegment;
             this.handlers[ODataSegmentKinds.UnboundFunction] = this.HandleUnboundFunctionPathSegment;
+            this.handlers[ODataSegmentKinds.Function] = this.HandleBoundFunctionPathSegment;
```

Comment:
Better add a static readonly action in this class probably called `NoOpHandler` whose body contains nothing and set it to both bound and unbound function handlers to make the code less and clear.